jetty-ee10/jetty-ee10-servlet/src/test/java/org/eclipse/jetty/ee10/servlet/DefaultServletTest.java

```diff
@@ -3456,35 +3452,35 @@ protected void service(HttpServletRequest req, HttpServletResponse resp) throws
         context.getServletHandler().addServlet(indexServlet);
         context.getServletHandler().addServletMapping(indexMapping);
 
-        Path pathTest = MavenTestingUtils.getTestResourcePath("pathTest");
+        Path docroot = MavenTestingUtils.getTestResourcePath("docroot");
```

Comment:
I was expecting to see the `ResourceServlet` used in this test, as I thought we want the `DefaultServlet` to be pretty much always mapped to `/` and anything else should be a `ResourceServlet` instead?

Reply:
Ultimately yes.  But we want to test that the DefaultServlet can be abused as it was previously (but now with new warnings), so we are testing it exactly the same as before.. which repeats some tests from ResourceServletTest.   Ultimately these can be removed when the warning turns into an ISE.
The ResourceServletTest has new more varied parameterized tests to show it can be used in more circumstances.